tracer/test/Datadog.Trace.Security.IntegrationTests/ApiSecurity/AspNetCore2ApiSecurity.cs

```diff
@@ -19,15 +19,15 @@ namespace Datadog.Trace.Security.IntegrationTests.ApiSecurity
     public class AspNetCore2ApiSecurityEnabled : AspNetCoreApiSecurity
     {
         public AspNetCore2ApiSecurityEnabled(AspNetCoreTestFixture fixture, ITestOutputHelper outputHelper)
-            : base(fixture, outputHelper, true, "AspNetCore2")
+            : base(fixture, outputHelper, true, true, "AspNetCore2")
```

Comment:
nit: bool parameters should use named parameters so it's easier to read what they're doing :)